pre_award/assess/assessments/templates/assessments/change_request_success_page.html

```diff
@@ -0,0 +1,52 @@
+{% extends "assess/base.html" %}
+{% from "assess/macros/sub_criteria_navbar.html" import navbar %}
+{% from "assess/macros/sub_criteria_heading.html" import sub_criteria_heading %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+
+
+{% set pageHeading -%}
+Request changes –
+{% if current_theme.id == sub_criteria.id %}
+  {{ sub_criteria.name }} – {{ sub_criteria.project_name }}
+{% else %}
+  {{ current_theme.name }} – {{ sub_criteria.name }} – {{ sub_criteria.project_name }}
+{% endif %}
+{% endset %}
+
+{% block header %}
+  {% include "assess/components/header.html" %}
+{% endblock header %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half-from-desktop">
+      <div class="govuk-panel govuk-panel--confirmation">
+        <h1 class="govuk-panel__title">
+          Your request for changes has been sent
+        </h1>
+      </div>
```

Comment:
Could we please use the component?
```
{{ govukPanel({
  "titleText": "Application complete",
  "html": "Your reference number<br><strong>HDJ2123F</strong>"
}) }}
```

https://design-system.service.gov.uk/components/panel/